Unity/Assets/Scripts/TutorialInstructions.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class TutorialInstructions : MonoBehaviour
+{
+    public GameObject InstrUI;
+    public Text instrText;
+
+    public void Show(string instr)
+    {
+        InstrUI.SetActive(true); // Set game object to active
+        instrText.text = instr;
+    }
+
+    public void Hide()
+    {
+        InstrUI.SetActive(false); // Set game object to not active
+        instrText.text = "";
+    }
+
+    private void OnControllerColliderHit(ControllerColliderHit hit)
+    {
+        if (hit.collider.name == "MoveInstrCollider")
+        {
+            Show("To move forward, use the arrow keys, WASD, or tilt the cotroller forward");
+        }
+        else if (hit.collider.name == "DuckInstrCollider")
+        {
+            Show("To duck, hold shift or duck while in frame of your webcam");
+        }
+        else if (hit.collider.name == "TurnInstrCollider")
+        {
+            Show("To turn, use 'z' and 'x' or tilt the controller left and right");

```

Comment:
Very minor but maybe can say "To turn left and right, use 'z' and 'x' or tilt the controller"

Likely also fine as is